kvdb-rocksdb/src/lib.rs

```diff
@@ -363,14 +363,7 @@ impl Database {
 			block_opts,
 			stats: stats::RunningDbStats::new(),
 			last_compaction: Mutex::new(Instant::now()),
-		};
-
-		// After opening the DB, we want to compact it.
-		//
-		// This just in case the node crashed before to ensure the db stays fast.
-		db.force_compaction()?;
```

Comment:
dq: This is not limited to archive nodes, but we have observed this while running archives?